integration/monitor_test.go

```diff
@@ -107,20 +104,17 @@ func TestMonitor(t *testing.T) {
 			t.Fatalf("CreateEpoch(_): %v", err)
 		}
 
-		mutResp, err := mutCli.PollMutations(ctx, env.Domain.DomainId, tc.queryEpoch)
-		if err != nil {
-			t.Fatalf("Could not query mutations: %v", err)
-		}
-
-		if err := mon.Process(mutResp); err != nil {
-			t.Fatalf("Monitor could not process mutations: %v", err)
+		domainID := env.Domain.DomainId
+		cctx, cancel := context.WithTimeout(ctx, 30*time.Millisecond)
```

Comment:
You could do the scoped local func / defer thing here too if you want.
It's all very small, so maybe not worthwhile at present, but could be if someone added code after `ProcessLoop` without thinking about the `cancel`.

Reply:
The reason I don't do it here is because I want to _block_, let `ProcessLoop` do it's thing,  timeout, then check the results. If I did it in parallel, I'd have a race condition.

Reply:
OK.  I can't see the original code with comments in context so we'll leave this.